README.md

```diff
@@ -642,22 +704,35 @@ Example:  `[showpass_calendar page="event-detail"]`  - This will tell the plugin
 
 The `page` parameter is `event-detail` in this example, but can be set to whatevet page you want.
 
-## 5.2. theme_dark Paramerter
+## 5.2. theme_dark Parameter
 Add `theme_dark="true"` to use a dark theme for the calendar instead of the default light theme.
 
-## 5.3. starting_date Paramerter
+## 5.3. starting_date Parameter
 Add `starting_date="1-12-2018"` use day-month-year to set a starting date for the calendar on initial load
 Format is day, month, year with no leading zeros
 
-## 5.4. use_widget Paramerter
+## 5.4. use_widget Parameter
 Add `use_widget="true"` if you want the ticket button to open the widget instead of redirecting to showpass or an event detail page use this
 
-## 5.5. tags Paramerter
+## 5.5. tags Parameter
 Add `tags="tag"` if you want to only display events with certain tags or categories. For best results use one word lowercase tags.
 
-## 5.6. hide_schedule Paramerter
+## 5.6. hide_schedule Parameter
 Add `hide_schedule="true"` if you do not want to display a daily schedule grouped by location.
 
+## 5.7. Recurring Events Parameter
+If you have recurring events, you can set whether or not you want to display them on the calendar or if you want to display only the main event and not each recurrence.
+
+`hide_children='true'` will not show any recurring events at all.
+
+`only_parents='true'` will show non-recurring events as well as the main recurring event. When using `only_parents='false'` this will show non-recurring events as well as all occurences of recurring events.
+
+By default `hide_children='false'` and `only_parents='false'`. In this case, non-recurring events are shown, as well as each occurence of a recurring event.
+
+## 5.8. Show Parameter
+
+Use this parameter for testing purposes. Using `show='all'` will show all events you have, including past events.
```

Comment:
I don't think it will show past events by default. Just events that are hidden, or visible to sellers only